### DISCUSSION TITLE
Privacy.com Integration

### DISCUSSION THREAD

**Discussion by AedenThomas** [simple-login/app#541] [Feature Requests]
It would be helpful if SimpleLogin would integrate with [Privacy card's IssuingAPI](https://privacy.com/card-issuing): [Lithic](https://lithic.com/) to generate virtual cards to mask our bank card.
This would be helpful so we can generate single use cards, set spend limits, pause and close cards, etc. when signing up for various services without disclosing our personal information.

**Comment by JoeBoll** [comment]
Doesn't [Privacy.com](https://privacy.com/) do this?

Not saying it's a bad idea, but I think most of the programmers on this project are from France, which means they can't even open an account on privacy/Lithic ... legally. [see screenshot] I'm from Canada which is also not allowed, so that means they'd also need to implement a system to keep people like me from using this US exclusive service.
<img width="621" alt="Screen Shot showing privacy.com's requirement to be a US citizen in order to sign up" src="https://user-images.githubusercontent.com/49468100/153582342-95ea4d3e-9549-4346-9b16-3f1d8eab0d7e.png">

**Reply by nguyenkims** [reply]
Yes privacy.com can't be used in Europe and we plan to not rely on privacy.com.

**Reply by c0nfigurati0n** [reply]
Yeah, sadly it isn't. Hopefully one day someone will come along and make a European and African version of privacy.com (and no [Revolut](https://www.revolut.com/) isn't the same as privacy.com).

**Reply by bravovictor42** [reply]
I am a privacy.com user and I would NOT, capitalizing for emphasis, not yelling, not use it. Their privacy policies and terms of service are not the best. MySudo is much better but also more expensive and more limiting for the number of cards and believe on US and Canada. Blur/Abine is the only one that is international; US, Canada, UK, South Africa, and 12 EU countries for masked calls alone. They offer a lot more but also have their own issues like masked cards being down for months while overhauling in past year. 

If anything, I would love to see masked phones and willing to pay for it. Blur works but it tries to do everything and everything becomes scrambled. Their card system is also masked. Privacy does mask cards, still needs zip code to validate and sometimes validation will require name. They also track location in name of security among other items per TOS. MySudo does not do this. Blur goes further but they also have a data breach on file from 2018 but at least confirmed they only have hashed passwords.

I am really hoping SimpleLogin fills in these gaps over time and would gladly pay the $100 that Blur asks for for having a decent solution even without the oddball items they throw in. Other options have been bought up in past couple years by larger companies. I very much hope this is the service that grows sustainably and can provide services that are severely lacking in options.
